src/metabase/models/pulse_channel.clj

```diff
@@ -127,7 +127,7 @@
    :schedule_type mi/transform-keyword
    :schedule_frame mi/transform-keyword})
 
-(methodical/defmethod t2/batched-hydrate [:default :recipients]
+(methodical/defmethod t2/batched-hydrate [:default :pulse-recipients]
```

Comment:
this is conflicting with notification :recipients method, it's suck that these key are global, i've some idea on how to make this better but let's do it later.